lib/iris/pandas.py

```diff
@@ -115,7 +115,8 @@ def as_cube(pandas_array, copy=True, calendars=None):
     _add_iris_coord(cube, "index", pandas_array.index, 0,
                     calendars.get(0, None))
     if pandas_array.ndim == 2:
-        _add_iris_coord(cube, "columns", pandas_array.columns, 1,
+        _add_iris_coord(cube, "columns", np.array([k for k in
```

Comment:
This would be better done as `pandas_array.columns.values`, which returns you a NumPy array of the column headers in one shot.

Reply:
good shout @dkillick, I'll get on it.

Reply:
Hmmm....

Reply:
@dkillick The problem here is that this method turns all the column headers into objects.  Now that seems fine to me, but @DavidSimonin seemed to think that they all should have been strings.  I'm not convinced that the headers should be strings at all, I think you can probably extract the string from the object, but I might have to dig a little deeper to determine the appropriate behaviour.

Reply:
@dkillick I reckon that should be it.  I removed and regenerated the cml file (self-serving as that seems...), and I'm banking on the tests passing and the fix being good now.